vkdgen.py

```diff
@@ -113,7 +113,9 @@ def convertTypeArray(typ, name):
 class DGenerator(OutputGenerator):
 	def __init__(self, errFile=sys.stderr, warnFile=sys.stderr, diagFile=sys.stderr):
 		super().__init__(errFile, warnFile, diagFile)
-	
+		self.enumConstants = []
+		self.funcNames = []
+

```

Comment:
> Why is reading the C headers a hassle?

Why should a D user have to when the same information is available in D modules?

> I personally prefer to use the specification as the reference, but I understand if you want to use the headers. But the D bindings are hardly any different than the C headers.

Have you at all looked at the erupted modules?? There is much more similarity between erupted modules and vulkan.h than there is between dvulkan and erupted.

Why are you arguing at all about that? There is nothing to loose but lot to gain.

> If you really want this, then submit a pull request changing it (and with only that change), and I will add it if it is implemented well enough.

I doubt that this would ever happen, it seems that your ego is a just to big barrier. Lets split ways, erupted will stay.